DataGenerator/src/test/resources/log4j2.properties

```diff
@@ -6,7 +6,7 @@ appender.console.type = Console
 appender.console.name = Console
 appender.console.target = SYSTEM_OUT
 appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = %m%n
+appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss,SSS}{UTC} %p %c{1.} [%t] %m%n
```

Comment:
Keeping thread identifiers is critical for multi-threaded code like this.  It's necessary to correlate log lines together.